dev/mapSTRING.R

```diff
@@ -0,0 +1,151 @@
+# Script to load STRING coexpression data and map it
```

Comment:
I can't recall whether Dr. Steipe said that he will upload it or not,
If it is uploaded by him through a different file, don't forget to remove yours!